Controller/Checkout.php

```diff
@@ -241,7 +241,10 @@ protected function getShippingAddressForQuote(array $data)
 
             $this->totalsCollector->collectAddressTotals($quote, $address);
             $quote->setTotalsCollectedFlag(false)->collectTotals();
-            $this->quoteRepository->save($quote);
+
+            if($data['block_key'] !== \Bread\BreadCheckout\Helper\Data::BLOCK_CODE_CHECKOUT_OVERVIEW){
```

Comment:
Noticed a problem with quote save on cart view page, so added this check, although it might be better to switch from !== to === since it's checking weather it's triggered from pdp page.

Reply:
There is a problem with qoute saving inside getShippingAddressForQuote, this part I added as fix, and it helped, but now when testing again problem is still here, I'm working on it.